src/shmem_synchronization.h

```diff
@@ -32,6 +32,8 @@ shmem_internal_quiet(shmem_ctx_t ctx)
 #ifdef USE_XPMEM
     ret = shmem_transport_xpmem_quiet();
     if (0 != ret) { RAISE_ERROR(ret); }
+#else
+    shmem_internal_membar();
 #endif
```

Comment:
I think we can delete the shmem_transport_xpmem_quiet/fence routines and just call shmem_internal_membar, right?  The latter routine already contains the checks for XPMEM.